lib/improver/nbhood.py

```diff
@@ -0,0 +1,161 @@
+# -*- coding: utf-8 -*-
```

Comment:
Really good point. We could add a test that loops over all Python files and checks the header. Will raise an issue.